src/net/reply.c

```diff
@@ -49,13 +40,13 @@ void _z_reply_data_free(_z_reply_data_t **reply_data) {
 
 z_result_t _z_reply_data_copy(_z_reply_data_t *dst, const _z_reply_data_t *src) {
     *dst = _z_reply_data_null();
+    dst->_tag = src->_tag;
```

Comment:
What the reason of this change?
In case of error will be nice to avoid unnecessary dst changes.

Reply:
Yeah so "no data" value of tag is `3` and not `0` so I was trying to avoid a `reply_data_init` function, but I guess that won't be avoided.